README.md

```diff
@@ -1,80 +1,133 @@
+  delete_after_merge: true
+
+# "update" defines how to keep open PRs up to date
+update:
+
+  # The "whitelist" and "blacklist" options here operate the same as described for the `merge` block.
+  whitelist:
+    labels: ["WIP", "Update Me"]
 ```
 
-The `mode` block specifies in which mode Bulldozer should operate. Available options:
-   - `whitelist` it will act on PRs that have a GitHub label with the text `MERGE WHEN READY`.
-   - `blacklist` it will not act on PRs that have a GitHub label with the text `DO NOT MERGE` or `WIP`.
-   - `pr_body` it will act on PRs that have in their body the text `==MERGE_WHEN_READY==`. This is useful for PRs from forks because usually people that fork the repo do not have write permissions on the source to attach the needed labels.
 
-The `strategy` field allows you to tune the merge strategy. Available options:
-  - `merge`: it will create a merge commit.
-  - `squash`: it will squash all the commits into one and will apply it on the target branch. With this merge strategy the user can specify the commit message ahead of time by putting a block like the following `==COMMIT_MSG== sample commit message ==COMMIT_MSG==` in the PR body.
-  - `rebase`: it will rebase all commits on top of the target branch.
+### Caveats and Notes
+
+If both `blacklist` and `whitelist` are specified, `bulldozer` will attempt to match on both. 
+In cases where both match, `blacklist` will take precedence.
 
-If the desired merge method is not allowed by the repository one will be chosen by default in the following order `squash, merge, rebase` from the ones allowed.
+The `merge_method` specifies the strategy that will be used to merge. Possible choices
+are `merge`, `squash`, and `rebase`. Specifying `squash` will allow for a further
+set of `squash_strategy` options, `pull_request_body`, `summarize_commits` and
+`empty_body` that will constitute the body of the merge commit message. 
 
-The `deleteAfterMerge` block specifies if the source branch of the PR should be deleted after the merge is performed.
+## Deployment
 
-If the repository has branch protection active but does not require PRs to have at least one reviewer Bulldozer will just merge the PR unless the creator of the PR assigned a reviewer explicitly. In that case it will wait for at least one APPROVED review
+`bulldozer` is easy to deploy in your own environment as it has no dependencies
+other than GitHub. It is also safe to run multiple instances of the server,
+making it a good fit for container schedulers like Nomad or Kubernetes.
 
-If the repository has branch protection active and enforces code reviews on PRs Bulldozer will wait until all status checks are green and at least one APPROVED review exists on the PR before merging it
+We provide both a Docker container and a binary distribution of the server. A
+sample configuration file is provided at `var/conf/bulldozer.yml`. We
```

Comment:
Is `var/conf/bulldozer.yml` still the correct path or did that move somewhere else?